definitions/io.lua

```diff
@@ -0,0 +1,13 @@
+-- starting to learn how to handle devices in code.
+
+return {
+  --  rename to midi_in ??
+  midi = {
+    vkb = 'Virtual Midi Keyboard', -- virtual keyboard | default
+    qmk = 'Ergodox EZ',
+    roland = 'Roland %- RD Series %- Port 1' -- my roland grand 88 key
```

Comment:
the comments here are all personalized as well. 

What I do for software is I keep an org file in emacs with information about my task I'm doing, and it helps keep me organized and keep track of information I need.